cmd/pke/app/phases/kubeadm/controlplane/kubeadm.go

```diff
@@ -214,6 +222,16 @@ apiServer:
     cloud-provider: "{{ .CloudProvider }}"
     cloud-config: /etc/kubernetes/{{ .CloudProvider }}.conf{{end}}
   extraVolumes:
+{{ if .WithAuditLog }}
+    - name: audit-log-file
+      hostPath: {{ .AuditLogFile }}
```

Comment:
is this a file or the directory for it? if i understand correctly, we should mount a directory for it because of the built-in log rotation

Reply:
nice catch, thanks